lisa/node.py

```diff
@@ -412,6 +416,9 @@ def find_partition_with_freespace(
                 else:
                     mountpoint = disk.mountpoint
 
+                if df.get_filesystem_available_space(mountpoint, True) < size_in_gb:
```

Comment:
What's the reason to check it again?

Reply:
updated

Reply:
Move the logic to be shared. The return part happens on both `if/else`, so it can be moved out from `if/else` too.

Reply:
updated